login.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+session_start();
+require 'db_connection.php';
+
+// --------------------------------------------------
+// POSTで送信されてきたIDがDBにあるか検索
+// --------------------------------------------------
+
+// 直接入れると参照できないためPHP変数に入れる
+$id = $_POST['id'];
+$password = $_POST['password'];
+
+try {
+    $db = new PDO(DSN, DB_USER, DB_PASS);
+    $sql = 'select id, name, password from users where id = :id';
+    $stmt = $db->prepare($sql);
+
+    //指定された変数名にパラメータをバインド
+    $stmt->bindParam(':id', $id);
+
+    //実行
+    $stmt->execute();
```

Comment:
```suggestion
    $stmt->execute([:id => $id]);
```
bindParam使用しなくてもこれでいけた気がします。

Reply:
いけたので修正しました